### ISSUE TITLE
Updating Entry Does Not Update Element

### ISSUE THREAD

**Issue by mattcdavis1** [craftcms/cms#2092]
### Description
I created a single with a URI of firm/philosopy (mispelled).  When i noticed the page wasn't found and realized it was because of the misspelling i updated the single uri to firm/philosophy.  The update update the `craft_sections` table but did not update the `craft_elements_sites` table so the old (misspelled) uri remained and i was unable to update it via the CP.

### Steps to reproduce

1. Create a single
2. Change the URI of the single
3. Craft will not route to the updated URI

### Additional info

- Craft version: Craft CMS 3.0.0-beta.30
- PHP version: 7.0.15
- Database driver & version: MySQL 5.5.57

**Comment by brandonkelly** [comment]
What is the URI format in your single’s section settings?

**Comment by mattcdavis1** [comment]
Here's a screenshot of the settings for the page in question: https://goo.gl/F6fErv

**Comment by brandonkelly** [comment]
Yeah so your URI is set from there – you can see it’s currently hard-coded as `firm/philosophy`. If you want the slug to be used in the URI, include a `{slug}` tag in there somewhere.

**Comment by brandonkelly** [comment]
Actually, I don’t think that’s possible. You just need to hard-code the URI you want in there.

**Comment by mattcdavis1** [comment]
That's the way i had it and it didn't work.  What are you saying to do different?

**Comment by brandonkelly** [comment]
Sorry, I misunderstood your original post. Will look into it…
